pkg/action/install_test.go

```diff
@@ -94,41 +95,30 @@ var _ = Describe("Install action tests", func() {
 	Describe("Install Action", Label("install"), func() {
 		var device, cmdFail string
 		var err error
-		var cmdline func() ([]byte, error)
 		var spec *v1.InstallSpec
 		var installer *action.InstallAction
 
 		BeforeEach(func() {
-			device = "/some/device"
+			os.RemoveAll("/tmp/test.img")
```

Comment:
oh, this call is now redundant (we call it again in the next line)

Reply:
ah yes, this was because I had some files from manual tests around and diskfs would fail if the file existed. Will remove!